ECMAScript/react/README.md

```diff
@@ -92,46 +108,17 @@ const MyComponent = () => {
 export default MyComponent
 ```
 
-##### useQuilttSettings
-
-```tsx
-import { QuilttSettings, useQuilttSettings } from '@quiltt/react'
-
-const App = () => {
-  // Wrap your application with QuilttSettingsProvider to provide the settings
-  return (
-    <QuilttSettings.Provider value={{ clientId: 'YOUR_CLIENT_ID' }}>
-      <MyComponent />
-    </QuilttSettings.Provider>
-  )
-}
-
-const MyComponent = () => {
-  const { clientId } = useQuilttSettings()
-
-  return (
-    <div>
-      <p>Quiltt Client ID: {clientId}</p>
-      {/* Rest of your component */}
-    </div>
-  )
-}
-
-export default App
-```
-
 ### React Providers
 
 The following providers are available from `@quiltt/react/providers`:
 
-- `QuilttAuthProvider`: A provider component for authenticating with Quiltt.
-- `QuilttProvider`: A provider component for initializing Quiltt.
-- `QuilttSettingsProvider`: A provider component for managing Quiltt settings.
+- `QuilttAuthProvider`: A provider for authenticating with Quiltt.
```

Comment:
`QuilttProvider` includes the other providers

```suggestion
```

Reply:
I am confused, I think Ruben wrote `QuilttAuthProvider` but you said "`QuilttProvider` includes the other providers".
Tell me more?

Reply:
The providers are nested. `QuilttProvider` wraps the other two so it's the primary one to use. We may even want to not export the sub-providers to make things simpler.

Reply:
I see, just make `QuilttProvider` the public interface.

Reply:
Since Providers are effectively components, I ended up reorganizing the documentation around 3 Components (Button, Container, Providers) and hooks, emphasizing the most important ones, and how they impact the `token` value being supplied to the Connector. 

I think this is much simpler now. We can consider un-exporting the sub-component Providers separately.